rusk-abi/tests/lib.rs

```diff
@@ -145,40 +215,61 @@ fn verify_proof() {
     let mut network = NetworkState::<MS>::default();
     network.register_host_module(rusk_mod);
 
-    let contract_id = network.deploy(contract).unwrap();
+    let contract = network.deploy(contract).unwrap();
```

Comment:
Revert this change, `network.deploy()` returns a `ContractId` this can be misleading.

Reply:
A contract is never used if not deployed to a network state so it can benefit from the storage implementations. Shadowing the original declaration (that is never going to be used) with a shorter name seems more meaningful since the only way to access a deployed contract is through its ID